modules/admin-ui/src/test/java/org/opencastproject/adminui/endpoint/AbstractEventEndpointTest.java

```diff
@@ -389,6 +389,7 @@ public void testGetAttachment() throws Exception {
   }
 
   @Test
+  @Ignore
```

Comment:
Adding a "//TODO: fix test"  will make it easier for other developers to see this ignore and address it in the future.

Reply:
Ignore it, its a temporary ignore until the code is complete :grin:

Reply:
Ok, I'm good with keeping the @ignore. Event though, IMHO, adding the //TODO makes it more obvious that this is an actual desired test which is temporarily under reconstruction and can be and *should* be implemented by someone, even if it is accidentally missed in this pull.